uninstall/docker.sh

```diff
@@ -1,2 +1,3 @@
-sudo apt remove -y docker.io docker-buildx
+sudo apt purge --auto-remove docker.io docker-buildx docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin docker-ce-rootless-extras
+sudo rm -f "${DOCKER_CONFIG:-$HOME/.docker}/cli-plugins/docker-compose"
```

Comment:
This is probably to cleanup from the previous iteration of this script (which installed in `~/.docker/cli-plugins`) correct?

Wondering if it could use a comment and/or instead to provide instructions to manually remove this version if it's present. The idea of the `~/.docker/cli-plugins` directory is to allow users to upgrade to a different version, so they may have installed it not through this script.

Reply:
Obviated by having uninstall scripts only target the latest incarnation of Omakub installs.

So it'll leave behind stuff you installed with a previous version of Omakub.